apps/frontend/src/pages/projects/libs/helpers/get-relative-date-label.helper.tsx

```diff
@@ -0,0 +1,38 @@
+import { getDifferenceInDays } from "~/libs/helpers/helpers.js";
+
+import { ColorClass, ColorThreshold } from "../enums/enums.js";
+
+type RelativeDateLabel = {
+	colorClass: string;
```

Comment:
```suggestion
	colorClass: ColorClass;
```

Reply:
Hm.. not sure here `colorClass: typeof ColorClass[keyof typeof ColorClass];`. Are we trying to enforce the colorClass to only accept values from the `ColorClass` object?